docs/book/component-gallery/orchestrators/airflow.md

```diff
@@ -87,6 +87,47 @@ This call will produce a `.zip` file containing a representation of your ZenML
 pipeline to the Airflow DAGs directory. From there, the local Airflow server
 will load it and run your pipeline (It might take a few seconds until the pipeline
 shows up in the Airflow UI).
+
+{% hint style="info" %}
+
+The ability to provision resources using the `zenml stack up` command
+is deprecated and will be removed in a future release. While it is still
+available for the Airflow orchestrator, we recommend following the
+steps to set up a local Airflow server manually.
+
+1. Install the `apache-airflow` package in your Python environment where ZenML is installed.
```

Comment:
We should do the same for the Label Studio stack component (plus the code warning).